cms/djangoapps/contentstore/views/course.py

```diff
@@ -525,9 +522,9 @@ def format_library_for_view(library):
     return render_to_response(u'index.html', {
         u'courses': list(courses_iter),
         u'in_process_course_actions': in_process_course_actions,
-        u'libraries_enabled': show_libraries,
+        u'libraries_enabled': LIBRARIES_ENABLED,
         u'libraries': [format_library_for_view(lib) for lib in libraries],
-        u'show_new_library_button': show_libraries and get_library_creator_status(user),
+        u'show_new_library_button': get_library_creator_status(user),
```

Comment:
Yup, it's the [first thing it checks](https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/views/library.py#L54)